### PR TITLE
Merge master to 1.0.x

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
 ## [Unreleased]
+### Added
+- Added platform support for:
+   * Debian 10 'Buster'
+   * Ubuntu 18.04 'Bionic'
+   * Fedora 28
+
+### Changed
+- Use `tar` labeled nodes to build tarballs
+- Renamed redhat-fips platform to redhatfips.
```
Comment (underscorgan) [inline CHANGELOG.md]: This looks wrong, should probably at least have a version thatalso got pulled in?

Comment (mwaggett) [inline CHANGELOG.md]: This is looking funky.. not sure how your PRs are playing with @sarameisburger's.

Comment (sarameisburger) [reply CHANGELOG.md]:
```
## [0.6.5] - 2018-04-16
### Added
- Added platform support for:
   * Debian 10 'Buster'
   * Ubuntu 18.04 'Bionic'
   * Fedora 28

### Changed
- Use `tar` labeled nodes to build tarballs
- Renamed redhat-fips platform to redhatfips.
```
Should be above 0.6.4

Comment (shrug) [reply CHANGELOG.md]: these changes are unreleased in this branch, right? they'll get included in the next release?

Comment (shrug) [reply CHANGELOG.md]: but the upcoming release will be the first in this branch to have those changes, yes? so they should go in 0.99.something?

Comment (shrug) [reply CHANGELOG.md]: better?

Comment (shrug) [reply CHANGELOG.md]: better?

**Pull request merged**